js/articleView.js

```diff
@@ -16,7 +27,7 @@ articleView.handleMainNav = function() {
     }
   })
   // show the blog articles on page load
-  $('nav .tab:second').click();
+  // $('nav .tab:second').click();
 };
 
 // hide some of the content
```

Comment:
You should rework the setTeasers function and the click handler: it's not a standard or good practice to have the click handler outside of a function like this. And it is calling setTeasers: maybe that can be done in a different way?